src/consts.js

```diff
@@ -68,54 +68,109 @@ export const META_ORIGINS = {
+        displayName: 'Node.js 14 on Alpine Linux',
         prePull: true,
     },
     {
-        name: 'apify/actor-node-chrome',
-        displayName: 'Node.js 12 + Chrome on Debian',
+        name: 'apify/actor-node-puppeteer-chrome',
+        displayName: 'Node.js 14 + Puppeteer + Chrome on Debian',
         copyChown: 'myuser:myuser',
         prePull: true,
     },
     {
-        name: 'apify/actor-node-chrome-xvfb',
-        displayName: 'Node.js 12 + Chrome + Xvfb on Debian',
+        name: 'apify/actor-node-playwright-chrome',
+        displayName: 'Node.js 14 + Playwright + Chrome on Debian',
+        copyChown: 'myuser:myuser',
+        prePull: true,
+    },
+    {
+        name: 'apify/actor-node-playwright-firefox',
+        displayName: 'Node.js 14 + Playwright + Firefox on Debian',
+        copyChown: 'myuser:myuser',
+        prePull: true,
+    },
+    {
+        name: 'apify/actor-node-playwright-webkit',
+        displayName: 'Node.js 14 + Playwright + WebKit on Debian',
+        copyChown: 'myuser:myuser',
+        prePull: true,
+    },
+    {
+        name: 'apify/actor-node-playwright',
+        displayName: 'Node.js 14 + Playwright + All Browsers on Ubuntu',
         copyChown: 'myuser:myuser',
         prePull: true,
     },
 
     // Beta:
     {
-        name: 'apify/actor-node-basic:beta',
-        displayName: 'BETA: Node.js 12 on Alpine Linux',
+        name: 'apify/actor-node',
+        displayName: 'BETA: Node.js 14 on Alpine Linux:beta',
     },
     {
-        name: 'apify/actor-node-chrome:beta',
-        displayName: 'BETA: Node.js 12 + Chrome on Debian',
+        name: 'apify/actor-node-puppeteer-chrome:beta',
+        displayName: 'BETA: Node.js 14 + Puppeteer + Chrome on Debian',
         copyChown: 'myuser:myuser',
     },
     {
-        name: 'apify/actor-node-chrome-xvfb:beta',
-        displayName: 'BETA: Node.js 12 + Chrome + Xvfb on Debian',
+        name: 'apify/actor-node-playwright-chrome:beta',
+        displayName: 'BETA: Node.js 14 + Playwright + Chrome on Debian',
+        copyChown: 'myuser:myuser',
+    },
+    {
+        name: 'apify/actor-node-playwright-firefox:beta',
+        displayName: 'BETA: Node.js 14 + Playwright + Firefox on Debian',
+        copyChown: 'myuser:myuser',
+    },
+    {
+        name: 'apify/actor-node-playwright-webkit:beta',
+        displayName: 'BETA: Node.js 14 + Playwright + WebKit on Debian',
+        copyChown: 'myuser:myuser',
+    },
+    {
+        name: 'apify/actor-node-playwright:beta',
+        displayName: 'Node.js 14 + Playwright + All Browsers on Ubuntu',
         copyChown: 'myuser:myuser',
     },
 
     // Deprecated:
-    // TODO: Keep the for some time and then migrate acts to recommended images.
+    // These are here because we made breaking changes in the client that could break existing single file actors.
+    // We will get a rid of this along with the whole single file logic.
     {
-        name: 'apify/actor-node-puppeteer',
```

Comment:
Shouldn't we keep these just renamed to deprecated? Otherwise, the building will break if somebody use it.

Reply:
`apify/actor-node-puppeteer` images were deprecated 3 years ago. I think it's safe to remove them.

Reply:
How about a quick query to see if somebody is still building with them?

Reply:
I did the query and there were only some free users using it with non-production actors. It seems that they stumbled upon the image in the list and were trying it out. => Better remove it.